Security/SessionDownloadStrategy.php

```diff
@@ -14,12 +14,16 @@
 use Sonata\MediaBundle\Model\MediaInterface;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Session\SessionInterface;
 use Symfony\Component\Translation\TranslatorInterface;
 
```

Comment:
i would add an author tag here, too, because you reworked a lot of this file

what do you think @greg0ire